python/hsfs/connection.py

```diff
@@ -147,6 +147,16 @@ def get_feature_store(self, name: str = None):
         Defaulting to the project name of default feature store. To get a
         Shared feature stores, the project name of the feature store is required.
 
+        !!! example "How to get feature store instance"
+
+            ```python
+            import hopsworks
+
+            project = hopsworks.login()
+
+            fs = project.get_feature_store()
```

Comment:
import hsfs
conn = hsfs.connection()
fs = conn.get_feature_store()

Reply:
Add the hsfs code as an alternative and always add the imports
'
import hopsworks
project = hopsworks.login()
fs = project.get_feature_store()
'
or
'
import hsfs
conn = hsfs.connection()
fs = conn.get_feature_store()
'